### DISCUSSION TITLE
빌드잇에서 가지고 와서 다른 scene에 추가하려고 하는 거 어떻게 하나요?

### DISCUSSION THREAD

**Discussion by yunah99** [naverz/zepeto-studio-kor#1287] [5. Q&A]
유니티에서 QR Code 생성을 했는데 애플로는 잘 들어가지는 반면 제 핸드폰(안드로이드) 화면에는 그냥 보라색 창 만 뜹니다.

혹시 이거 해결하는 방법 아시나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 개발자 디스코드가 공식적으로 오픈되었습니다. 
https://discord.gg/6Z9dBM7H3e
새로운 질문은 디스코드를 이용해 주시면 감사하겠습니다.
